### DISCUSSION TITLE
Domovea Sequenzen in HomeKit einbinden

### DISCUSSION THREAD

**Discussion by kevinkandziora** [brutella/hkknx-public#258] [Q&A] [answered]
Hallo zusammen,

ich versuche schon seit einigen Tagen die Sequenzen aus Domovea (Hager easy) ins HomeKit über die hkknx bridge zu übertragen. Leider weiß ich nicht, ob Sequenzen, die in Domovea erstellt worden sind überhaupt Gruppenadressen zugewiesen bekommen bzw. wo diese zu finden sind.
Ich habe auch schon über die ETS und einem Plugin meine KNX Installation ausgelesen, aber ich habe leider keine Sequenzen gefunden.
Hat jemand vielleicht das gleiche Problem und konnte es lösen?

Vielen Dank im Voraus!

**Comment by 0x636b** [comment, marked as answer]
Hey.

Sequenzen sind Automationen innerhalb der Visu. Diese besitzen keine Gruppenadressen. Das was du vllt. meinst sind Szenen. Du kannst mithilfe der ETS Szenennummern an verschiedene Aktorkanäle knüpfen und diese dann anhand der Nummer aufrufen. Das funktioniert dann auch in HKKNX / Home.

**Reply by kevinkandziora** [reply]
Vielen Dank. Ich denke, ich habe es jetzt verstanden.
